doc/releases/quincy.rst

```diff
@@ -5,6 +5,771 @@ Quincy
 Quincy is the 17th stable release of Ceph. It is named after Squidward
 Quincy Tentacles from Spongebob Squarepants.
 
+
+
+v17.2.8 Quincy
+==============
+
+This is the eighth, and expected to be last, backport release in the Quincy series. We recommend
+that all users update to this release.
+
+Notable Changes
+---------------
+
```

Comment:
I suggested for `Notable Changes` heading to be moved, not for the second (duplicate) one to be added.  Remove the first one:
```suggestion
```